tests/Feature/FavoritesControllerTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use App\Models\City;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class FavoritesControllerTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testCityCanBeAddedToAndRemovedFromFavorites(): void
```

Comment:
I think I will divide this test to two separate - one should check if city can be added to favorites and another one to check is city can be removed.

Reply:
Now it is separated